### ISSUE TITLE
Cannot read property 'spritesheetURL' of undefined

### ISSUE THREAD

**Issue by Viveur** [flobacher/SVGInjector2#8]
The standard code given under usage did not work for me, and gave the error in the title.

I had to include spritesheetURL: false in the options, ie:

```
// Elements to inject
var mySVGsToInject = document.querySelectorAll('img.inject-me');
// Do the injection
new SVGInjector({spritesheetURL: false}).inject(mySVGsToInject);
```

**Comment by flobacher** [comment]
should be fixed with https://github.com/flobacher/SVGInjector2/releases/tag/v2.0.29

**Comment by Viveur** [comment]
Yep, appears to be fixed in 2.0.29. Thanks

**Comment by flobacher** [comment]
great! thx for the feedback!
